doc/adr-001-otp2-update-eval.md

```diff
@@ -0,0 +1,106 @@
+
+
+### OTP1 vs OTP2 Assessment
+
+Noting first that there is an [available OTP1 --> OTP2 migration guide](https://docs.opentripplanner.org/en/latest/Version-Comparison/#migration-guide) as well as [detailed comparison page](https://docs.opentripplanner.org/en/v2.3.0/Version-Comparison/#summary) in the offical Open Trip Planner docs.
+
+#### OTP 1.4
+**Pros:**
+- Current implementation of core features
+- OTP1 maintenance status appears to have long-term support
+    - From the OTP repo CHANGELOG: “OTP1 is essentially a legacy product that will receive bug and stability fixes to the extent that they can be readily backported from OTP2”.
+- Graph build process already refined	
+    - Feed sources, data configurations, and necessary commands already setup using ansible roles
+    - Official docs detail graph hot reloading as a unique OTP1 feature, allowing us to update transit feeds without re-building the graph each month (the most time-intensive part of monthly deployments) which could be worth investigating as part of automating data deployment.
+
+**Cons:**
+- Does not support updated GTFS data format, which affects SEPTA transit feed
+    - Minimal changes (converting trolleybuses) to allow graph to build, however sometimes other formatting errors in GTFS data feeds and manually validating can be time intensive 
+- We are one minor version behind, should still upgrade to capture latest bugfixes
+    - While it will be minor update changes we don’t have other projects with Java VM environments to develop a strong familiarity and we haven’t updated OTP in years so potentially unexpected lift
+    - From release notes: “Version 1.5 of OpenTripPlanner was released just before version 2.0. It is intended to serve as a final release integrating any bugfixes backported from the 2.x development as the 1.x track moves into legacy/maintenance status.”
+- OTP1 will remain but OTP2 is recommended practice
+    - At the time of writing, the OTP official documentation notes that OTP2 is not intended to supersede OTP1 and the online recommendation for new users is to “try out OTP2 and switch to OTP1 if they need features that are not available in OTP2”.
+
+
+#### OTP 2
+
+**Pros:**
+- Actively developed with large community of contributers and is stable version for the foreseeable future
+- Aligned with GTFS and GBFS feed formats and likely will maintain alignment
+- More efficient graph building and performant routing 
+
+**Cons**
+- Does not support core features
+    - OTP2 has removed isochrone generation entirely, so we will not be able to generate the travel shed geometry which we use to find destinations within a defined travel radius. This critically affects the “Explore” mode
+- Sweeping changes to routing API
+    - Does not support "bcycle" rental bike updator out of the box, which we currently use for indigo rental bike information like station updates
```

Comment:
Is `bcycle` a typo?

Reply:
It's actually the name of the manufacturers behind Indego bikes and stations (and other cities' bike shares)! It seems like they[ maintain the GBFS spec](https://www.bcycle.com/gbfs) and then the operator (indigo) opts in, so we configure the otp server with our "bcycle indigo" url that pings for bike share updates.